src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -542,8 +543,9 @@ def msOutputConsumer(self):
                     msg = "%s General error from pipeline. Err: %s. " % (pipeLineName, str(ex))
                     msg += "Will retry again in the next cycle."
                     self.logger.exception(msg)
-                    break
-                wfCounters += 1
+                    workflowname = docOut.get("_id", "none")
+                    self.alertGenericError("consumer", workflowname, msg, str(ex), str(docOut))
```

Comment:
Instead of writing "consumer" here, perhaps pass the variable `self.mode`

Reply:
uuuuh nice suggestion, thanks!